recipes/libi2c/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+make BUILD_DYNAMIC_LIB=1 BUILD_STATIC_LIB=0 install
```

Comment:
does `-j${CPU_COUNT}` work?

Reply:
I am not sure as the `Makefile` of the project is manually written and not generated, I can try. However the package is so small (it builds in like ~15 seconds), so I do not think it plays a big role here.

Reply:
I am not sure it is actually using more cores, but in any case the compilation works fine.